python/src/KrigingResult_doc.i.in

```diff
@@ -219,6 +219,79 @@ condCov : :class:`~openturns.CovarianceMatrix`
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::KrigingResult::getConditionalMarginalCovariance
+"Compute the expected covariance of the Gaussian process on a point (or several points).
+
+
+Available usages:
+    getConditionalMarginalCovariance(x)
+
+    getConditionalMarginalCovariance(sampleX)
+
+Parameters
+----------
+x : sequence of float
+    The point :math:`\vect{x}` where the conditional mean of the output has to be evaluated.
```

Comment:
"conditional mean" should be replaced by "conditional covariance"

Reply:
Indeed, will be fixed in a new PR